test/service/test_spec.ml

```diff
@@ -62,7 +62,7 @@ let expected_linux_spec =
 (* Create testable Sexp for Alcotest. *)
 let sexp = Alcotest.testable Sexplib0__Sexp.pp_hum Sexplib0__Sexp.equal
 
-let test_macos_spec () =
+let test_macos_spec _ () =
```

Comment:
I don't understand why these tests need to use Alcotest LWT now. They do not require any LWT.

Reply:
This avoids splitting  the tests in LWT and none.

Reply:
And to fit the tests in two categories `test/service` and `test/web`.